packages/content-gen/docs/MESHY_API.md

```
@@ -0,0 +1,379 @@
+rigging (input_task_id OR model_url + texture_image_url)
+    │
+    ▼ result.rigged_character_glb_url
+animations (rig_task_id + action_id)
+    │
+    ▼ result.animation_glb_url
+```
+
+---
+
+*Last Updated: 2025-01-15*
+*Source: https://docs.meshy.ai/en/api/*
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)